code/game/objects/items/pet_carrier.dm

```diff
@@ -350,4 +350,140 @@
 	if(..())
 		name = "[initial(name)] ([target])"
 
+
+
+// Same code at the bluespace cumjar I mean pet carrier but wont break when thrown allowing re-use.
+/obj/item/pet_carrier/bcball
+	name = "bluespace capture ball"
+	desc = "An elaborate bluespace containment device able to hold and release any monsters or catboys placed inside when thrown."
+	open = FALSE //starts closed so it looks better on menus
+	icon_state = "bcball"
+	item_state = "bcball"
+	lefthand_file = ""
+	righthand_file = ""
+	max_occupant_weight = MOB_SIZE_HUMAN //can fit people, like a bluespace bodybag!
+	load_time = 40 //loading things into a jar takes longer than a regular pet carrier
+	entrance_name = "lid"
+	w_class = WEIGHT_CLASS_SMALL //it's a small ball
+	throw_speed = 3
+	throw_range = 7
+	max_occupants = 1 //far less than a regular carrier or bluespace bodybag, because it can be thrown to release the contents
+	allows_hostiles = TRUE //can fit hostile creatures, with the move resist restrictions in place, this means they still cannot take things like legions/goliaths/etc regardless
+	has_lock_sprites = FALSE //bcball doesn't show the regular lock overlay
+	custom_materials = list(/datum/material/glass = 1000, /datum/material/bluespace = 4000, /datum/material/diamond = 4000)
+	escape_time = 200 //equal to the time of a bluespace bodybag
+	alternate_escape_time = 100
+
+	///gas supply for simplemobs so they don't die
+	var/datum/gas_mixture/occupant_gas_supply
+	///level until the reagent gets INGEST ed instead of TOUCH
+	var/sipping_level = 150
+	///prob50 level of sipping
+	var/sipping_probably = 99
+	///chem transfer rate / second
+	var/transfer_rate = 5
+
+/obj/item/pet_carrier/bcball/Initialize()
+	. = ..()
+	create_reagents(300, OPENCONTAINER, DEFAULT_REAGENTS_VALUE) //equivalent of bsbeakers
```

Comment:
I dont think we need the reagent part?

Reply:
Thought about it, mixed on keeping or not. Haven't fully decided but I'm leaning towards removing adding reagents